### PR TITLE
Two postponed TODOs in recent PRs...

### PR DESCRIPTION
**Pull request by rocky**
* Move eval functions to mathics.eval.numbers.numbers
* Move "no_doc" after imports

### CODE REVIEW

File: mathics/eval/numbers/numbers.py [hunk 1]
```diff
@@ -140,25 +222,11 @@ def cancel(expr):
             return expr
 
 
-def cancel(expr):
```
Comment (mmatera) [inline mathics/eval/numbers/numbers.py:143]: What happens with this function? Is it not needed anymore?

**Review by mmatera — APPROVED** [review]
Apart from my comment about the apparently unused `cancel` function, LGTM

**Review by davidar — APPROVED** [review]
LGTM

File: mathics/eval/numbers/numbers.py (hunk 1 above)
Comment (rocky) [reply mathics/eval/numbers/numbers.py:143]: The function is duplicated. This is easier to see in the master branch right now: 

https://github.com/Mathics3/mathics-core/blob/master/mathics/eval/numbers/numbers.py#L122-L149

image

Comment (mmatera) [reply mathics/eval/numbers/numbers.py:143]: Thanks for the clarification!

**Pull request merged**